### ISSUE TITLE
Classification banner and Footer on the same page causes an accessibility violation

### ISSUE THREAD

**Issue by MI6-255** [mi6/ic-ui-kit#317]
## Summary of the bug
Classification banner and Footer on the same page causes an accessibility violation due to both having the "contentinfo" role. 

## 🪜 How to reproduce
Tell us the steps to reproduce the problem:
1. Place the Classification banner and Footer onto a page
2. Run an accessibility test
3. See the notice for having two "contentinfo" on one page (due to footer assuming this role)

## 🧐 Expected behaviour
Expected to not have any accessibility violations when used together.

## Additional info
The solution to this is to change the Classification banner to a <section> component instead of a div, and removing the "contentinfo" role. Tests will need to be updated.
